isos/bootstrap-staging.sh

```diff
@@ -106,6 +106,7 @@ yum_cached -c $cache -u -p $PKGDIR install \
     libtirpc \
     grep \
     haveged \
+    dbus \
```

Comment:
By the way the other alternative might be calling the syscall directly from the tether to set the hostname.

But in every case not sure why systemd photon package does not depend on dbus (or whether it's OK to not to have it). It looks like dbus is a hard requirement for systemd to work.